pkg/diagnostics/metrics.go

```diff
@@ -68,5 +68,11 @@ func InitMetrics(appID, namespace string, rules []config.MetricsRule) error {
 
 	// Set reporting period of views
 	view.SetReportingPeriod(DefaultReportingPeriod)
-	return utils.CreateRulesMap(rules)
```

Comment:
Isn't this change functionally same and more verbose than a direct return statement?

Reply:
Reverted it. I do find that a bit clearer to read but not worth the diff.